moderation/db.py

```diff
@@ -17,4 +17,6 @@ class ModelBase(ModeratedModelBase, base.ModelBase):
 
 
 class ModeratedModel(six.with_metaclass(ModelBase, base.Model)):
-    pass
+
+    class Meta:
+        abstract = True
```

Comment:
If this isn't here, then Django creates a database table for `moderation.ModeratedModel`.